cl_sii/dte/constants.py

```diff
@@ -209,3 +209,19 @@ def emisor_is_vendedor(self) -> bool:
     @property
     def receptor_is_vendedor(self) -> bool:
         return self.is_factura_compra
+
+
+class XmlSchemasVersionEnum(enum.Enum):
```

Comment:
@glarrain the directory also contains the XML schemas for RTC and RCV, there are even changes in this PR that apply to the validation of the AEC schemas.

Reply:
Then the enum shouldn't belong in `cl_sii/dte/constants.py`, right?

Reply:
> Then the enum shouldn't belong in `cl_sii/dte/constants.py`, right?

Good catch. Where do you think it will fit best? `cl_sii/base/constants.py`?

Reply:
> > Then the enum shouldn't belong in `cl_sii/dte/constants.py`, right?
> 
> Good catch. Where do you think it will fit best? `cl_sii/base/constants.py`?

`cl_sii/base/constants.py` looks good to me.